### PR TITLE
Assistant next step

### PR DESCRIPTION
**Pull request by kevinkowa**
### Summary

This is a hand written change to update next_step in a dialog node to null.

### CODE REVIEW

**Comment by apaparazzi0329** [comment]
This is looking good. I would add an integration test to ensure that a previously populated `next_step` is now empty after applying the new parameter

**Review by apaparazzi0329 — APPROVED** [review]
LGTM

**Comment by apaparazzi0329** [comment]
My only concern right now is how are we going to document this for other users since it will not be on the official Assistant docs

**Comment by apaparazzi0329** [comment]
Maybe add a more descriptive comment for the parameter and what it does

File: assistant/src/main/java/com/ibm/watson/assistant/v1/Assistant.java [hunk 1]
```diff
@@ -2248,6 +2248,12 @@ public void setVersion(final String version) {
           com.ibm.cloud.sdk.core.util.GsonSingleton.getGson()
               .toJsonTree(updateDialogNodeOptions.newNextStep()));
     }
+    if (updateDialogNodeOptions.deleteNextStep() != null && updateDialogNodeOptions.deleteNextStep()) {
+      contentJson.add(
+              "next_step",
+              com.ibm.cloud.sdk.core.util.GsonSingleton.getGson()
+                      .toJsonTree(null));
```
Comment (mediumTaj) [inline assistant/src/main/java/com/ibm/watson/assistant/v1/Assistant.java:2255]: Does this actually put `null` into `next_step` in the json? want to make sure gson isn't disregarding null values

File: assistant/src/main/java/com/ibm/watson/assistant/v1/model/UpdateDialogNodeOptions.java [hunk 2]
```diff
@@ -731,4 +747,15 @@ public Boolean newDisambiguationOptOut() {
...
+  /**
+   * Gets the deleteNextStep.
+   *
+   * <p>Set nextStep to null
+   *
+   * @return the deleteNextStep
+   */
+  public Boolean deleteNextStep() {
+    return deleteNextStep;
+  }
```
Comment (mediumTaj) [inline assistant/src/main/java/com/ibm/watson/assistant/v1/model/UpdateDialogNodeOptions.java:760]: this is a lot of handwritten changes

**Review by mediumTaj — COMMENTED** [review]
Lets talk about this PR during parking lot.

File: assistant/src/main/java/com/ibm/watson/assistant/v1/Assistant.java (hunk 1 above)
Comment (kevinkowa) [reply assistant/src/main/java/com/ibm/watson/assistant/v1/Assistant.java:2255]: yes, this is the output from `nc -l` 
```
{"dialog_node":"Test2d28f9c6e-8864-4bbb-8ab8-81d18b740ce9","description":"Updated description of Test79155c70-05aa-47a3-b1e2-5557109f89b3","next_step":null}
```

Comment (apaparazzi0329) [reply assistant/src/main/java/com/ibm/watson/assistant/v1/Assistant.java:2255]: It does. We looked at the final request sent to the service and it `inserts` null into the request
